ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/KVTest.scala

```diff
@@ -237,6 +235,8 @@ object KVTest {
       result <- submit(subm)
     } yield result
 
+  val minMRTDelta: Duration = Duration.ofSeconds(1)
```

Comment:
what was this needed for? configuration? Any thoughts on how we should handle timing out other submission types?

Reply:
Right now:
- Only the config change request still has an MRT, party allocations and package uploads don't have any kind of timeout.
- There is no limit to how large the user can set the MRT to.
- The submission should be rejected by the ledger if record time > MRT.

I'm not sure how much value this has in practice however:
- We're going to remove heartbeats, and record times are not monotonically increasing anymore. So the user will not know *exactly* when the the request timed out.
- The config change request has a generation number that can be used to prevent race conditions.
- I would imagine that the ledger config is only changed very infrequently (if at all), and only by a small set of trusted admins.

So maybe we should remove the MRT on config change requests? @dajmaki @gerolf-da